### PR TITLE
[libpcap] Fix pkgconfig libs and include path

### PR DESCRIPTION
**Pull request by ayum**
Fixing pkgconfig's "Libs" field because of wrong package name with a new one line patch.

Fixing pkgconfig's "includedir" path for debug build. For this purpose add vcpkg_fixup_pkgconfig cmake function. May be it is usefull and may be considered for inclusion in scripts/cmake?

### CODE REVIEW

**Comment by cenit** [comment]
I did something simpler here: https://github.com/cenit/vcpkg/blob/61586a3c1ad909cb6f38bcd28c4e9c967964fc39/scripts/cmake/vcpkg_fixup_pkgconfig_targets.cmake

It might really be worth the effort to prepare something interesting for everybody. Pkgconfig cannot be forgotten anymore, imho

**Comment by NancyLi1013** [comment]
Hi @almikhayl thanks for this PR.
Could you also bump the version in CONTROL file?
Since this PR is related with something that @cenit mentioned above.
It needs to do further review.

**Comment by almikhayl** [comment]
@JackBoosY 
Why vcpkg-osx-PR build failed? I can't view it, because "Build not found".

**Comment by NancyLi1013** [comment]
@almikhayl 
`vcpkg-osx-PR` has deprecated. we have changed to `vcpkg-osx-PR-test`.
So we only need to pay attention to the results on `vcpkg-osx-PR-test`.

**Comment by traversaro** [comment]
As a new general vcpkg `vcpkg_fixup_pkgconfig` has been recently added and it has been used in https://github.com/microsoft/vcpkg/pull/10731, probably this PR has been superseed by https://github.com/microsoft/vcpkg/pull/10731 ?

**Comment by NancyLi1013** [comment]
@almikhayl 

`vcpkg_fixup_pkgconfig()` has been added to vcpkg. This PR might be updated now.

In addition, could you please help confirm if this PR have done some similar work with #10731?

**Comment by almikhayl** [comment]
@NancyLi1013 

Updated PR to use new vcpkg_fixup_pkgconfig.

As for #10731 . These are two quite different PRs. This PR is only small fix PR and use vcpkg_fixup_pkgconfig PR. Very much unrelated to each other.

File: ports/libpcap/portfile.cmake [hunk 1]
```diff
@@ -34,16 +37,10 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/libpcap RENAME copyright)
```
Comment (NancyLi1013) [inline ports/libpcap/portfile.cmake:38]:
```suggestion
file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

File: ports/libpcap/CONTROL [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Source: libpcap
-Version: 1.9.1
+Version: 1.9.1-1
 Description: A portable C/C++ library for network traffic capture
 Homepage: https://www.tcpdump.org/
```
Comment (NancyLi1013) [inline ports/libpcap/CONTROL:4]:
```suggestion
Homepage: https://www.tcpdump.org/
Supports: linux
```

File: ports/libpcap/portfile.cmake [hunk 3]
```diff
@@ -18,6 +20,7 @@ vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${SOURCE_ARCHIVE_PATH}
     REF 1.9.1
+    PATCHES 0001-fix-package-name.patch
```
Comment (NancyLi1013) [inline ports/libpcap/portfile.cmake:23]:
```suggestion
    PATCHES 
        0001-fix-package-name.patch
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
@almikhayl 
Thanks for your update.
It looks good to me now.

**Comment by traversaro** [comment]
> As for #10731 . These are two quite different PRs. This PR is only small fix PR and use vcpkg_fixup_pkgconfig PR. Very much unrelated to each other.

I totally agree.  #10731 is a superset of this PR, and if this PR can be merged before it would be trivial to rebase #10731 on top of it.

**Comment by dan-shaw** [comment]
Thanks for the PR!

**Pull request merged**